tests/licensedcode/data/datadriven/lic4/2523-utils-merge_package.json.yml

```diff
@@ -0,0 +1,4 @@
+license_expressions:
+  - mit
+  - mit
```

Comment:
Just curious, why is the same license listed twice here?

Reply:
@pombredanne maybe you can answer this